packages/components/package.json

```diff
@@ -25,40 +26,27 @@
     "lint:typescript": "../../scripts/check-typescript"
   },
   "devDependencies": {
-    "@babel/runtime": "^7.12.5",
-    "date-fns": "^2.16.1",
     "nebenan-eventproxy": "^1.0.0",
     "nebenan-form": "^10.0.0",
-    "nebenan-helpers": "^7.1.0",
     "nebenan-keymanager": "^1.0.1",
-    "nebenan-ui-kit": "^5.1.0",
     "node-sass-glob-importer": "^5.3.2",
-    "prop-types": "^15.7.2",
-    "qs": "^6.9.4",
-    "react": "^16.14.0",
-    "react-dom": "^16.14.0",
-    "react-load-script": "^0.0.6",
     "react-redux": "^7.2.2",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
-    "redux": "^4.0.5",
-    "sass-functions": "^2.0.0",
```

Comment:
Are you really sure we can remove all those devDeps? I think most of them are deps of peerDependencies - `npm i` doesn't cry?

Reply:
will do the package updates later when we get typescript export ready